amplitude/types/event_storage.go

```diff
@@ -1,7 +1,9 @@
 package types
 
 type EventStorage interface {
-	Push(event *Event)
-	Pull() []*Event
```

Comment:
Is "Push" and "Pull" common naming in Go? If not, could we rename these methods to match Android-Koltin (writeEvent, readEvents, etc). I would like to keep the interfaces as similar as possible.

https://github.com/amplitude/Amplitude-Kotlin/blob/8c46250e3ee957e22f97ad5d6e957e12ebe82272/core/src/main/java/com/amplitude/core/utilities/FileStorage.kt

Reply:
Based on Python implementation. We need to get and remove items, so `Pull` looks better for me. `Push` for symmetry.